styleguide/derek/examples/homepage.ejs

```diff
@@ -2,182 +2,190 @@
 <%- partial("../_partials/_mainnav") %>
 
 <!-- Hero -->
-  <div class="container-fluid homepage-hero">
-    <div class="container standard-padding-full">
-      <div class="col-md-8">
-        <h2>Expertise comes standard.<br/> Choose your Managed Cloud.</h2>
-        <p class="left white">Trusted by more than 60% of the Fortune 100. Serving 300,000+ businesses globally. A leader in the Gartner Magic Quadrant for cloud-enabled managed hosting.</p>
-        <div class="row">
-          <div class="col-lg-4 col-md-6">
-            <a href="#chat" class="button lead">Speak with a specialist</a>
-          </div>
+<div class="container-fluid homepage-hero homepageHero">
+  <div class="container standard-padding-full">
+    <div class="col-md-8">
+      <h2 class="homepageHero-title">Expertise comes standard.<br/> Choose your Managed Cloud.</h2>
+      <p class="left white homepageHero-subtext">Serving more than half of the Fortune 100. A leader in the Gartner Magic Quadrant for cloud-enabled managed hosting.</p>
+      <div class="row">
+        <div class="col-lg-5 col-md-6">
+          <a href="#chat" class="button lead">Speak with a specialist</a>
         </div>
       </div>
     </div>
   </div>
+</div>
 
-  <!-- Hosting Options -->
-  <div class="container standard-padding" id="hosting-options">
-    <div class="row">
-      <div class="col-md-12 lead">
-        <h3>We're the largest managed cloud provider, offering you expertise across the world's leading clouds</h3>
-      </div>
+<!-- Hosting Options -->
+<div class="container standard-padding hostingOptions" id="hosting-options">
+	<div class="row half-padding-full">
```

Comment:
These tab sizes are larger than rest of file.  Can we use 2 spaces across the board?